server/public/style.css

```diff
@@ -0,0 +1,8 @@
+.card-flex {
+  flex-basis: 30%;
+}
```

Comment:
Using the appropriate column class as mentioned in my previous comment should eliminate the need for this.